esx_service/vmdk_ops_test.py

```diff
@@ -532,10 +549,296 @@ def test_vmdkop_authorize(self):
         # remove the tenant
         error_info = self.auth_mgr.remove_tenant(tenant1.id, False)
         self.assertEqual(error_info, None)
-        error_info = self.auth_mgr.remove_volumes_from_volume_table(tenant1.id)
-        self.assertEqual(error_info, None)        
-                                                  
+        error_info = self.auth_mgr.remove_volumes_from_volumes_table(tenant1.id)
+        self.assertEqual(error_info, None)
+
+class VmdkTenantTestCase(unittest.TestCase):
+    """Unit test for VMDK ops for multi-tenancy"""
+    non_tenant_vol_name = "non_tenant_vol1"
+
+    # tenant1 info
+    tenant1_name = "test_tenant1"
+    vm1_name = 'test_vm1'
+    tenant1_vol1_name = 'tenant1_vol1'
+    tenant1_vol2_name = 'tenant1_vol2'
+    tenant1_vol3_name = 'tenant1_vol3'
+    vm1_config_path = None
+
+    # tenant2 info
+    tenant2_name = "test_tenant2"
+    vm2_name = 'test_vm2'
+    tenant2_vol1_name = 'tenant2_vol1'
+    tenant2_vol2_name = 'tenant2_vol2'
+    tenant2_vol3_name = 'tenant2_vol3'
+    vm2_config_path = None
+    
+    datastore_name = None
+    datastore_path = None
+    
+
+    def setUp(self):
+        """ Setup run before each test """
+        logging.debug("VMDKTenantTest setUp path =%s", path)
+        
+        if (not self.datastore_name):
+            datastores = vmdk_utils.get_datastores()
+            datastore = datastores[0]
+            if (not datastore):
```

Comment:
this looks suspicious. Is there a case where non-empty array with None as the first element is returned by get_datastores() ?